src/redux/actions/projectActions.ts

```diff
@@ -146,18 +146,15 @@ export function deleteProject(project: IProject)
         const appState = getState();
         const projectService = new ProjectService();
 
-        // Lookup security token used to decrypt project settings
-        const projectToken = appState.appSettings.securityTokens
-            .find((securityToken) => securityToken.name === project.securityToken);
-
-        if (!projectToken) {
-            throw new AppError(ErrorCode.SecurityTokenNotFound, "Security Token Not Found");
+        let sourceConnection = appState.connections.find(a => a.name === project.sourceConnection.name);
+        if (!sourceConnection) {
+            toast.error(strings.errors.connectionNotExistError.message, { autoClose: false });
+        }
+        else {
+            const decryptedProject = { ...project, sourceConnection };
+            await projectService.delete(decryptedProject);
+            dispatch(deleteProjectAction(decryptedProject));
```

Comment:
we can only remove project from FoTT tool, but make sure we didn't delete the project file, otherwise, someone can delete others project file with this method.